packages/react/src/provider/use-context-mutator.ts

```diff
@@ -0,0 +1,23 @@
+import { OpenFeature, EvaluationContext } from '@openfeature/web-sdk';
+
+type DomainContextMutator = (domain: string, updatedContext: EvaluationContext) => Promise<void>; 
+
+/**
+ * 
+ * A hook for accessing context mutating functions.
+ * 
+ */
+export function useContextMutator() {
+
+  async function mutateContext(domainOrUpdatedContext: string | EvaluationContext, updatedContextOrUndefined?: EvaluationContext): Promise<void> {
```

Comment:
Starting to think about how to write some tests. Looking at the readme I see you can set a context by domain but also have a global context when no domain is set:

https://github.com/open-feature/js-sdk/tree/main/packages/react#openfeatureprovider-context-provider

Could you confirm these testing requirements:

`test('should modify context for the domain if one is present')`

`test('should modify context for global scope when no domain is found')`

Reply:
test code here. let me know if it would be good to have a test for the global context case as well

https://github.com/open-feature/js-sdk/pull/1031/commits/a7090f099cce3be28565de935cd2119bfeb41372